### ISSUE TITLE
Survey/Scan

### ISSUE THREAD

**Issue by arvian1401** [rayocta303/ESP8266-Range-Extender-GUI#2]
Error/Padahak udh Bgus. Syang Cuma Error di Scannny aja. Please Update Gan.

**Comment by kmunanya** [comment]
nah, seperti kurang stabil juga,tapi versi 2.0 lebih stabil dari versi 1.0,buatkan versi 3.0 nya juga sir hhe
